plugins/reach/lib/model/data/VendorSignLanguageCatalogItem.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ *  VendorSignLanguageCatalogItem
+ * @package plugins.reach
+ * @subpackage model
+ */
+
+class VendorSignLanguageCatalogItem extends VendorCatalogItem
+{
+    public function applyDefaultValues()
+    {
+        $this->setServiceFeature(VendorServiceFeature::SIGN_LANGUAGE);
+    }
+
+    public function isDuplicateTask(entry $entry)
+    {
+        return false;
+    }
+
+    public function requiresEntryReady()
+    {
+        return true;
+    }
+
+    /**
+     * @param $object
+     * @return kSignLanguageVendorTaskData|null
+     */
+    public function getTaskJobData($object)
+    {
+        if($object instanceof asset)
```

Comment:
@TalDubovKaltura  what determines what the $object in this case will be? An automated process, or a manual action?

Reply:
@TalDubovKaltura  so please change the check here to instanceof flavorAsset

Reply:
Fixed